fastplotlib/widgets/histogram_lut.py

```diff
@@ -161,9 +162,10 @@ def _calculate_histogram(self, data):
             hist, edges = np.histogram(data_ss, bins=self._nbins)
 
         # used if data ptp <= 10 because event things get weird
-        # with tiny world objects due to  floating point error
+        # with tiny world objects due to floating point error
         # so if ptp <= 10, scale up by a factor
-        self._scale_factor: int = max(1, 100 * int(10 / np.ptp(data_ss)))
+        data_interval = edges[-1] - edges[0]
+        self._scale_factor: int = max(1, 100 * int(10 / data_interval))
```

Comment:
@kushalkolar True. But I think the bigger reason why this is important is that if you do np.histogram on a dataset with constant value (i.e. np.zeros((Dim1, Dim2)), np.ptp will return 0, leading to divide by 0. But the edges[-1] - edges[0] avoids this issue since np.histogram computes a reasonable interval.